tests/test_blend.py

```diff
@@ -115,7 +118,7 @@ def test_fit_extended_source(self):
 
         # Fit the model
         blend.fit(100)
-        assert blend.it == 13
+        assert blend.it == 15
```

Comment:
I find these hard-coded numbers a little disturbing. I am pretty sure these are going to change everytime we change the algorithm. Maybe we could put `assert blend.it < 16` or 20 just to be on the safe side. This way, if a new minimisation makes convergence faster by even just one iteration, it won't crash the tests. `16` is a good way to make sure that further alterations of the code will only improve convergence.

Reply:
I like this idea. By the end of the year this should be removed completely because I hope to have QA testing in LSST by then so we'll be able to track exactly how long blends are taking on a much larger (and more useful) scale. In the meantime I like your proposed solution.

Reply:
I decided to go with 20, since there is a possibility that a future improvement (like deconvolution) might make this one example slightly worse while improving convergence in general.